pkg/minikube/assets/assets.go

```diff
@@ -0,0 +1,2621 @@
+// Code generated by go-bindata. (@generated) DO NOT EDIT.
```

Comment:
please remove these files these not meant to be in this PR (this is old asset files that been deleted)